app/_gateway_entities/workspace.md

```diff
@@ -16,14 +16,129 @@ tier: enterprise
+
+
+{% mermaid %}
+flowchart LR
+    subgraph Workspace1 [Workspace-1]
+        A(Team A's - Service)
+        B(Route)
+    end 
+
+    subgraph Workspace2 [Workspace-2]
+        C(Team B's - Service)
+        D(Route)
+
+    end
+
+    subgraph Workspace3 [Workspace-3]
+        E(Team C's - Service)
+        F(Route)
+    end
+    subgraph Gatewayorg [Gateway Organization]
+    Workspace1
+    Workspace2
+    Workspace3
+    end
+ 
+{% endmermaid %}
+
+
+
+### How does {{site.base_gateway}} resolve entity conflicts between Workspaces?
+
+Routing rules are configured at the data plane level. The data plane routes client traffic based on the configuration applied across all Workspaces. Configuring entities related to routing, such as [Gateway Services](/gateway/entities/service/) and [Routes](/gateway/entities/route/), alter the client traffic routing behavior of the data plane, but {{site.base_gateway}} will always attempt to ensure that routing rules don't contain conflicts. 
+
+To route traffic to the appropriate Workspace, {{site.base_gateway}} uses a conflict detection algorithm.
+
+When a Service or Route is **created** or **modified**, the {{site.base_gateway}} Router checks for the existence of that object before allowing the operation to proceed in this order:
+
+* If the Service or Route created is totally unique and does not match an existing entity, the new entity is created. 
+* If an existing Service or Route object that matches the one being created is found, a `409 Conflict` error is returned. 
+* If an equivalent Service or a Route is found in a different Workspace, the new entity is created.
+* If an equivalent Service or Route is found in a different Workspace, and the host is a wildcard: 
+  * If the host field matches in both workspaces, a `409 Conflict` error is returned.
+  * If the host field does not match, the new entity can be created.
+  * If the host is an absolute value: Issue a `409 Conflict` error.
+
+ 
+
+## Roles, groups, and permissions
+
+Because Workspaces allow users to control {{site.base_gateway}} entities in isolation, users must have the correct permissions to configure a particular Workspace. Users will require either a Super Admin or Admin role to configure Workspaces. 
+
+The following table details which Workspace permissions each Admin role has:
+
+| Permission | Super Admin | Admin |
+|-----------|---------------|-------|
+| Manage entities within the specified Workspace |  ✅  |  ✅  |
```

Comment:
can we use yaml syntax for this? I can cherry-pick https://github.com/Kong/developer.konghq.com/pull/199/commits/d558a4e4e1e19afe0ef8f19ca95fda49b4877183

Reply:
Yeah i'll do that.